IntegrationTests/IntegrationScenarios/Scenarios/RUM/RUMResourcesScenarioTests.swift

```diff
@@ -110,7 +110,12 @@ class RUMResourcesScenarioTests: IntegrationTests, RUMCommonAsserts {
         )
 
         // Get RUM Sessions with expected number of View visits and Resources
-        let rumRequests = try rumServerSession.pullRecordedRequests(timeout: dataDeliveryTimeout) { requests in
+        //
+        // RUMM-3151: Timeout of 30s makes the test flaky, increasing to over 30s
+        // allow the session to end. I suspect the view event debounce mechanism
+        // to postpone the end of the session when a resource completes after the
+        // stop of its parent view.
+        let rumRequests = try rumServerSession.pullRecordedRequests(timeout: dataDeliveryTimeout * 2) { requests in
             try RUMSessionMatcher.singleSession(from: requests)?.hasEnded() ?? false
         }
```

Comment:
@ncreated I think that the reason for our flakiness on `develop` as well.

The `try RUMSessionMatcher.singleSession(from: requests)?.hasEnded()` stays false for more than 30s because there is an active view. After 30s, we get a request with a view update making the view inactive.

I think it's due to our debounce mechanism, it seems that the view update is sent after 30s because the view was previously stopped but with a pending resource 🤔

Reply:
That wasn't it
<img width="1380" alt="image" src="https://user-images.githubusercontent.com/6815992/235162724-f0687564-a05a-41b3-8839-7fe98cd98ffb.png">